molecule/rbac-mtls-provided-ubuntu/molecule.yml

```diff
@@ -182,6 +182,8 @@ provisioner:
 
       kafka_broker:
         ldap_config: |
+          listener.name.broker.plain.sasl.server.callback.handler.class=io.confluent.security.auth.provider.ldap.LdapAuthenticateCallbackHandler
```

Comment:
ldap callback handler only works w sasl plain, this scenario has mtls on all the listeners, so probably the wrong place. 

here's my one pager for implementation in CFK: https://confluentinc.atlassian.net/wiki/spaces/CLUSTER/pages/2684093338/Sasl+Plain+LDAP

there were some gotchas for me

Reply:
Thanks so much @domenicbove , I'll go through your one pager and will make the required changes.